test/integration/util/kube_apiserver/events_test.go

```diff
@@ -48,7 +48,7 @@ func TestKubeEvents(t *testing.T) {
 	require.NotNil(t, core)
 
 	// Ignore potential startup events
-	_, _, initresversion, err := apiclient.LatestEvents("")
+	_, _, initresversion, err := apiclient.LatestEvents("0")
```

Comment:
Forcing "0" instead of "", as we only want to use the APIServer's local cache.
if "" is passed, it panics:
```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x10452d0]
```
The problem being with `resVersionCached, cachedResErr := strconv.Atoi(*latestResVersion)` [below](https://github.com/DataDog/datadog-agent/pull/1009/commits/3509c94605cbabb7c3bf67a77fc8505a2bc11a19#diff-8f1c08effc368ad3aa91fdd0007818dbR68). Although it returns 0 for the conversion (expected), it returns an error.

The solution I found would be to add `*latestResVersion=0` here:
https://github.com/DataDog/datadog-agent/pull/1009/commits/3509c94605cbabb7c3bf67a77fc8505a2bc11a19#diff-8f1c08effc368ad3aa91fdd0007818dbR41.

Thoughts @xvello, @JulienBalestra ?

Reply:
lgtm